requirements.txt

```diff
@@ -12,4 +12,4 @@ category_encoders>=2.2.2
 statsmodels >= 0.12.2
 imbalanced-learn>=0.8.0
 pmdarima==1.8.0
-sktime>=0.5.3;python_version<"3.9"
+sktime>=0.5.3,<0.7.0;python_version<"3.9"
```

Comment:
Will revert later :)

Reply:
Was this related to the PR? Just curious :)